### PR TITLE
add screenshot of low RMAU warning to WhatsApp guide

### PR DESCRIPTION
**Pull request by rtanglao**
as per new low RMAU warning in:
https://github.com/matrix-org/matrix-hosted/pull/3755

### CODE REVIEW

File: src/Add-WhatsApp-Bridge.md [hunk 1]
```diff
@@ -16,8 +16,13 @@ Click the `Integrations` tab  and if you have more than one server, select the s
 1. Choose `WhatsApp` from the list of available Bridges
 ![](images/wa-matrix-choose-bridge.png)  
 
-1. Enter the maxium number of users in `Maximum WhatsApp users` e.g. `20` and then click `Purchase`  
-![](images/wa-enter-number-users-click-purchase.png)
+1. Enter the maximum number of users in `Maximum WhatsApp users`  
+Please note:  **this is the maximum number of WhatsApp users who actually send messages over the bridge each month. You are only billed for the number of WhatsApp users who are active. Once you exceed the maximum, then the bridge will be disabled until you increase the maximum.**  
+If you enter less than `5`, you will get a warning  
+![](images/wa-low-rmau-warning.png)  
+If you enter a number `5` or greater in `Maximum WhatsApp users` e.g. `20`, you will not see a warning
```
Comment (jaller94) [inline src/Add-WhatsApp-Bridge.md:23]:
```suggestion
If you enter `5` or greater in `Maximum WhatsApp users`, you will not see a warning.  
```

File: src/Add-WhatsApp-Bridge.md [hunk 2]
```diff
@@ -16,8 +16,13 @@ Click the `Integrations` tab  and if you have more than one server, select the s
 1. Choose `WhatsApp` from the list of available Bridges
 ![](images/wa-matrix-choose-bridge.png)  
 
-1. Enter the maxium number of users in `Maximum WhatsApp users` e.g. `20` and then click `Purchase`  
-![](images/wa-enter-number-users-click-purchase.png)
+1. Enter the maximum number of users in `Maximum WhatsApp users`  
+Please note:  **this is the maximum number of WhatsApp users who actually send messages over the bridge each month. You are only billed for the number of WhatsApp users who are active. Once you exceed the maximum, then the bridge will be disabled until you increase the maximum.**  
+If you enter less than `5`, you will get a warning
```
Comment (jaller94) [inline src/Add-WhatsApp-Bridge.md:21]:
```suggestion
If you enter less than `5`, you will get a warning.  
```

File: src/Add-WhatsApp-Bridge.md [hunk 3]
```diff
@@ -16,8 +16,13 @@ Click the `Integrations` tab  and if you have more than one server, select the s
 1. Choose `WhatsApp` from the list of available Bridges
 ![](images/wa-matrix-choose-bridge.png)  
 
-1. Enter the maxium number of users in `Maximum WhatsApp users` e.g. `20` and then click `Purchase`  
-![](images/wa-enter-number-users-click-purchase.png)
+1. Enter the maximum number of users in `Maximum WhatsApp users`
```
Comment (jaller94) [inline src/Add-WhatsApp-Bridge.md:19]:
```suggestion
1. Enter the maximum number of users in `Maximum WhatsApp users`.  
```

File: src/Add-WhatsApp-Bridge.md [hunk 4]
```diff
@@ -16,8 +16,13 @@ Click the `Integrations` tab  and if you have more than one server, select the s
...
 
-1. Enter the maxium number of users in `Maximum WhatsApp users` e.g. `20` and then click `Purchase`  
-![](images/wa-enter-number-users-click-purchase.png)
+1. Enter the maximum number of users in `Maximum WhatsApp users`  
+Please note:  **this is the maximum number of WhatsApp users who actually send messages over the bridge each month. You are only billed for the number of WhatsApp users who are active. Once you exceed the maximum, then the bridge will be disabled until you increase the maximum.**  
+If you enter less than `5`, you will get a warning  
+![](images/wa-low-rmau-warning.png)  
+If you enter a number `5` or greater in `Maximum WhatsApp users` e.g. `20`, you will not see a warning  
+![](images/wa-enter-number-users-click-purchase.png)  
+Once you have entered `Maximum WhatsApp users`, click `Purchase` (remember you can always go back to this step and increase the maximum number of `WhatsApp` users if you need more in the future).
```
Comment (jaller94) [inline src/Add-WhatsApp-Bridge.md:25]:
```suggestion
Once you have entered `Maximum WhatsApp users`, click `Purchase` (remember you can always go back to this step and increase the maximum number of WhatsApp users if you need more in the future).
```

File: src/Add-WhatsApp-Bridge.md (hunk 2 above)
Comment (jaller94) [inline src/Add-WhatsApp-Bridge.md:21]: I would prefer all sentences to end with a period. But this style without a period on list items wasn't introduced in this PR.
```suggestion
1. Enter the maximum number of users in `Maximum WhatsApp users`.  
Please note:  **this is the maximum number of WhatsApp users who actually send messages over the bridge each month. You are only billed for the number of WhatsApp users who are active. Once you exceed the maximum, then the bridge will be disabled until you increase the maximum.**  
If you enter less than `5`, you will get a warning.  
```

Comment (rtanglao) [reply src/Add-WhatsApp-Bridge.md:21]: yeah i think maybe Twilight and I informally decided to have a period at the end of the sentence except there is a screenshot? I'll discuss with twilight

Comment (Twi1ightSparkle) [reply src/Add-WhatsApp-Bridge.md:21]: I'm equally fine with either, as long as we're consistent

File: src/Add-WhatsApp-Bridge.md (hunk 1 above)
Comment (rtanglao) [reply src/Add-WhatsApp-Bridge.md:23]: yes i agree this is better wording, thanks. resolving

File: src/Add-WhatsApp-Bridge.md (hunk 4 above)
Comment (rtanglao) [reply src/Add-WhatsApp-Bridge.md:25]: @Twi1ightSparkle @jaller94 What's a better way or a more standard way of marking up names of Apps like `WhatsApp`

File: src/Add-WhatsApp-Bridge.md (hunk 2 above)
Comment (rtanglao) [reply src/Add-WhatsApp-Bridge.md:21]: let's go with no periods at the end of the sentence when the sentence ends before a screenshot.

File: src/Add-WhatsApp-Bridge.md (hunk 4 above)
Comment (rtanglao) [reply src/Add-WhatsApp-Bridge.md:25]: yeah maybe it's better to only write it in a code block when you are actually clicking on the word `WhatsApp` That appears to be how  @Twi1ightSparkle wrote the Slack guide.

Comment (rtanglao) [reply src/Add-WhatsApp-Bridge.md:25]: ok i will only use a code block when actually clicking on WhatsApp! make sense on third thought :-)

Comment (rtanglao) [reply src/Add-WhatsApp-Bridge.md:25]: done so resolving

**Pull request merged**